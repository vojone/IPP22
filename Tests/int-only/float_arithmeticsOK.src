<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode22">
	<instruction order="10" opcode="DEFVAR">
		<arg1 type="var">GF@test1</arg1>
	</instruction>
    <instruction order="15" opcode="DEFVAR">
		<arg1 type="var">GF@test2</arg1>
	</instruction>
    <instruction order="20" opcode="MOVE">
		<arg1 type="var">GF@test1</arg1>
        <arg2 type="float">1.5</arg2>
	</instruction>
    <instruction order="30" opcode="MOVE">
		<arg1 type="var">GF@test2</arg1>
        <arg2 type="float">1.5</arg2>
	</instruction>
    <instruction order="40" opcode="ADD">
		<arg1 type="var">GF@test1</arg1>
        <arg2 type="var">GF@test1</arg2>
        <arg3 type="var">GF@test2</arg3>
	</instruction>
    <instruction order="50" opcode="SUB">
		<arg1 type="var">GF@test1</arg1>
        <arg2 type="var">GF@test1</arg2>
        <arg3 type="float">-3.0</arg3>
	</instruction>
    <instruction order="60" opcode="MUL">
		<arg1 type="var">GF@test1</arg1>
        <arg2 type="var">GF@test1</arg2>
        <arg3 type="float">2.0</arg3>
	</instruction>
    <instruction order="70" opcode="DIV">
		<arg1 type="var">GF@test1</arg1>
        <arg2 type="var">GF@test1</arg2>
        <arg3 type="float">0x1.0000000000000p+2</arg3>
	</instruction>
    <instruction order="80" opcode="WRITE">
		<arg1 type="var">GF@test1</arg1>
	</instruction>
</program>