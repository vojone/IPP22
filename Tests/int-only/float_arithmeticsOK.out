0x1.8000000000000p+1